R/new_ode_model.R

```diff
@@ -293,8 +297,27 @@ new_ode_model <- function (model = NULL,
       if(!file.exists(new_folder)) {
         dir.create(new_folder)
       }
-      file.copy(from = templ_folder, to = new_folder,
-                overwrite = TRUE, recursive = TRUE, copy.mode = FALSE)
+      file.copy(
+        from = templ_folder,
+        to = new_folder,
+        overwrite = TRUE,
+        recursive = TRUE,
+        copy.mode = FALSE
+      )
+
+      ## copy definition JSON, if model created from definition file.
+      if(!is.null(definition) && !is.null(attr(definition, "path"))) {
+        dir.create(file.path(new_folder, "inst"))
+        copy_result <- file.copy(
+          from = attr(definition, "path"),
```

Comment:
I think it's slightly nicer to copy the original JSON file into the package, rather than the serialized & unserialized JSON. That's why I'm not just doing `jsonlite::toJSON(definition)` and saving to file. But open for other opinions.